### PR TITLE
fix: change board notice apis to v2

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
This PR includes many refactoring.
Please review in order by commits!

### CODE REVIEW

File: apps/web/src/services/info/pages/NoticeDetailPage.vue [hunk 1]
```diff
@@ -253,7 +244,7 @@ export default {
                         { k: 'created_at', v: createdAt, o: '<t' },
                     ]);
                 }
-                const { results } = await SpaceConnector.client.board.post.list({
+                const { results } = await SpaceConnector.clientV2.board.post.list<PostListParameters, PostListResponse>({
```
Comment (piggggggggy) [inline apps/web/src/services/info/pages/NoticeDetailPage.vue]: 🥲

**Review by piggggggggy — APPROVED** [review]
LGTM!!

File: apps/web/src/services/info/pages/NoticeDetailPage.vue (hunk 1 above)
Comment (yuda110) [reply apps/web/src/services/info/pages/NoticeDetailPage.vue]: 🥰

**Review by yuda110 — APPROVED** [review]
LGTM!!

**Pull request merged**